client/blocks/reader-full-post/index.jsx

```diff
@@ -235,7 +235,14 @@ export class FullPostView extends React.Component {
 	attemptToSendPageView = () => {
 		const { post, site } = this.props;
 
-		if ( post && post._state !== 'pending' && site && site.ID && ! this.hasSentPageView ) {
+		if (
+			post &&
+			post._state !== 'pending' &&
+			site &&
+			site.ID &&
+			! site.is_error &&
+			! this.hasSentPageView
```

Comment:
Probably worth adding a few comments too to explain the more unusual checks. Out of context I'm not sure folks realize there's a reader site for example that may have an `is_error` prop.